### PR TITLE
CFD-392 Organisations & Email domains : Add organisation entity + settings

### PR DESCRIPTION
**Pull request by HelenaEksler**
#584

### CODE REVIEW

**Comment by HelenaEksler** [comment]
selection_301
selection_302

File: capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module [hunk 1]
```diff
@@ -103,8 +103,10 @@ function c4m_content_group_node_update($node) {
   }
 
   // Status now and before update.
-  $status_new = $node->c4m_og_status[LANGUAGE_NONE][0]['value'];
-  $status_original = (isset($node->original->c4m_og_status[LANGUAGE_NONE][0]['value']))
+  $status_new = isset($node->original->c4m_og_status)
```
Comment (HelenaEksler) [inline capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module]: ``$status_new = isset($node->original->c4m_og_status)`` => ``$status_new = isset($node->c4m_og_status)``

**Comment by HelenaEksler** [comment]
@nader77 Review this, please.

**Comment by HelenaEksler** [comment]
@nader77 Can you review this, please?

**Comment by nader77** [comment]
@HelenaEksler Looks good, @zero2one PR is ready for code review.

**Comment by zero2one** [comment]
Can you replace the dummy data icons for EC/EEAS & EU by:

### EC/EEAS

ec-logo


### EU

eu-logo

**Comment by HelenaEksler** [comment]
selection_306

**Comment by HelenaEksler** [comment]
@zero2one Ready.

**Pull request merged**